plugins/modules/zos_copy.py

```diff
@@ -2576,6 +2578,8 @@ def run_module(module, arg_def):
         if volume:
             dest_data_set["volumes"] = [volume]
 
+    copy_member = is_member(dest)
```

Comment:
Adding the member validation into the zos_copy module will not make a big impact since is just a regex validation.

Reply:
And is a validation over the dest, that doesn't need to be done on controller.